Gemfile

```diff
@@ -44,7 +44,9 @@ gem "libxml-ruby", platforms: :ruby
 gem "connection_pool", require: false
 
 # for railties app_generator_test
-gem "bootsnap", ">= 1.1.0", require: false
+gem "bootsnap", ">= 1.4.0", require: false
+
+gem "zeitwerk", ">= 1.0.0" if RUBY_ENGINE == "ruby"
```

Comment:
Need `platform:` here; `if` and Gemfiles don't mix well

Reply:
Oh yes.